README.md

```diff
@@ -47,7 +47,7 @@ flowchart LR
 
 4. Raise a Pull Request:
 
-- Click `Pull Requests` (which is the third option at the top of this page after the options `Code` and `Issues`).
+- Click `Pull Requests` (which is the second option at the top of this page after the options `Code` and `Issues`).
```

Comment:
I think it was fine before: the PRs tab is the third option and not the second option.
Please change it back to how it was before.

```suggestion
- Click `Pull Requests` (which is the third option at the top of this page after the options `Code` and `Issues`).
```